pyaedt/hfss.py

```diff
@@ -72,6 +72,9 @@ class Hfss(FieldAnalysis3D, object):
         Port number of which start the oDesktop communication on already existing server.
         This parameter is ignored in new server creation. It works only on 2022R2.
         Remote Server must be up and running with command `"ansysedt.exe -grpcsrv portnum"`.
+    aedt_process_id : int, optional
+        Only used when new_desktop_session == False, specifies by process id which instance
+        of electronics desktop to point PyAEDT at.
```

Comment:
@camuher  I like this approach. Anyway you should add the aedt_process_id to any of the tools (q2d, q3d, maxwell3d etc...) otherwise you will be able to initialize the app using process id only from Hfss

Reply:
@maxcapodi78 thank you for the quick reply. As specified in my latest commit I've added it to almost all the tools(and parent classes as needed) which makes for a uniform variable across the board.  The only one I couldn't add it to was siwave whose __init__ differs significantly from the structure used by the other tools.  I hope this is sufficient to address your request.

Reply:
@camuher 
you have to ignore siwave 
it is different approach.
Please merge from main to get a fix on UT and then I can approve the PR

Reply:
@maxcapodi78 I've merged to be up to date with main now.  I also addressed the error that caused the UT for rmxport and twinbuilder to fail.  I hadn't added the variable to their parent classes.  There were also miscellaneous edits to make the code more uniform with the style.

Reply:
@camuher I still see some style error. I suggest to install pre-commit on your environment.
pip install pre-commit
pre-commit install

It will automatically fix style errors on commit.

PS can you ping me on Teams? I'd like to know who you are :-)